shared/src/formationData.ts

```diff
@@ -421,6 +421,13 @@ export const incorporationLegalStructures: FormationLegalType[] = [
   "nonprofit",
 ];
 
+export const nonCorporationLegalStructures: FormationLegalType[] = [
```

Comment:
delete

Reply:
Agreed, we can infer this based on the information we already have. Having 2 competing lists creates a potential for errors.